horovod/spark/common/util.py

```diff
@@ -587,3 +587,23 @@ def get_dataset_properties(dataset_idx):
 
 def clear_training_cache():
     _training_cache.clear()
+
+
+def to_list(var, length):
+    if var is None:
+        return None
+
+    if not isinstance(var, list):
+        var = [var]
+
+    # If only one loss function or loss_constructor is passed by user, use
```

Comment:
Since this is in `common`, we should not reference loss and loss_constructors.

Reply:
addressed